dvc/command/imp_url.py

```diff
@@ -57,4 +60,10 @@ def add_parser(subparsers, parent_parser):
     import_parser.add_argument(
         "-f", "--file", help="Specify name of the DVC-file it generates."
     )
+    import_parser.add_argument(
+        "--remove-outs",
+        action="store_true",
+        default=False,
+        help="Confirm removal of existing files used as outputs",
```

Comment:
took me a while to understand the semantics. Let's do something like "Always remove existing outputs w/o asking for a confirmation data is not in cache and can be lost"